index.html

```diff
@@ -1,311 +1,321 @@
+                                    <span class="icone-about">
+                                        <img src="/img/js.png" alt="js-icone" class="fav html">
+                                        <span class="label">JavaScript</span>
+                                    </span>
+                                </li>
+                                <li class="container span-about">
+                                    <span class="icone-about">
+                                        <img src="/img/html.png" alt="html-icone" class="fav html">
+                                        <span class="label">HTML</span>
+                                    </span>
+                                </li>
+                                <li class="container span-about">
+                                    <span class="icone-about">
+                                        <img src="/img/ic-css.png" alt="CSS-icone" class="fav css">
+                                        <span class="label">CSS</span>
+                                    </span>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="frameworks-about">
+                            <p class="frameworks">Frameworks</p>
+                            <img src="/img/Disabled-2.png" alt="row-icone" class="row-icone">
+                        </li>
+
+                        <li class="skills-about">
+                            <p class="skills">Skills</p>
+                            <img src="/img/Disabled-2.png" class="row-icone" alt="row-icone">
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+
+        <section class="contact-form">
+            <div class="container desktop">
+            <div class="contact-content">
+                <h1 class="contact-me">Contact me</h1>
+                <p class="contact">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/xqkjaqoe" method="post">
+                <div class="name-container">
+                    <input name="name" type="text" placeholder="your full name" maxlength="30" required>
+                </div>
+                <br>
+                <div class="mail-container">
+                    <input name="email" type="email" placeholder="natajayanj14@gmail.com" required>
+                </div>
+                <br>
+                <div class="textarea-container">
+                    <textarea name="message" maxlength="500" placeholder="Write your message here" required></textarea>
+                </div>
+                <br>
+                <button type="submit">Get it Touch</button>
```

Comment:
- [OPTIONAL] Please style your button for hover and active as in the Figma template. It's a project requirement. As not required by the previous reviewer I consider this optional.